client/app.jsx

```diff
@@ -32,7 +31,10 @@ class App extends React.Component {
     }
 
     getId() {
-      const path = window.location.pathname;
+      var path = window.location.pathname;
+      if (path === '/') {
+        path = '1'
```

Comment:
Does this need to be '/1' or  just '1'?

Reply:
It actually doesn't matter since a few lines below that I regex the number out anyway.